api/classes/organizr.class.php

```diff
@@ -2663,7 +2663,7 @@ public function createFirstAdmin($username, $password, $email)
 			'order' => 1,
 			'category' => 'Unsorted',
 			'category_id' => 0,
-			'image' => 'plugins/images/categories/unsorted.png',
+			'image' => 'plugins/images/categories/fontawesome::question',
```

Comment:
you dont need the file path as you are using just an icon... either way - i will just update on backend as its not urgent